TACC/protx-dashboard
# Task/COOKS-341: refactor FigureCaption and ChartInstructions

## Overview: ##
Refactor figure caption code

## Related Jira tickets: ##

* [COOKS-341](https://jira.tacc.utexas.edu/browse/COOKS-341)

## Summary of Changes: ##
- Refactored the FigureCaption component to use `children` props instead of a prop using an array.
- Added unit test for FigureCaption component
- Refactored the table caption code in AnalyticsPredictiveTable to use the FigureCaption component

## Testing Steps: ##
1. Go to: [https://cep.test/protx/dash/analytics](https://cep.test/protx/dash/analytics)
2. Click on a county and verify that the chart and table captions are correctly displayed.

## UI Photos:

## Notes: ##
